src/components/Nav/Nav.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch, faTimes } from "@fortawesome/free-solid-svg-icons";
+import "./Nav.scss";
+
+class Nav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      searchInput: "",
+      popular: [],
+      showModal: false,
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      searchInput: e.target.value,
+    });
+  };
+
+  enterInsertValue = (e) => {
+    if (e.key === "Enter") {
+      this.props.history.push({
+        pathname: "/SearchTest",
+        state: { test: this.state.searchInput },
+      });
+    }
+  };
+
+  handleModal = () => {
+    this.setState({
+      showModal: true,
+    });
+  };
+
+  handleModalRemove = () => {
+    this.setState({
+      showModal: false,
+    });
+  };
+
+  componentDidMount() {
+    fetch("/data/SearchResultData.json")
+      .then((response) => response.json())
+      .then((data) => {
+        this.setState({
+          popular: data,
+        });
+      });
+  }
+
+  render() {
+    const { searchInput, showModal, popular } = this.state;
+    const { handleModal, handleChange, enterInsertValue, handleModalRemove } =
+      this;
+    return (
+      <div className="nav">

```

Comment:
```suggestion
      <nav className="nav">

```
nav 태그 사용해보는 건 어떨까요? 그리고 바로 아랫단의 nav 태그는 불필요하게 한겹 덮고 있는거 같아서 제거해주셔도 될거 같아요

Reply:
수정하였습니다!